stdlib/websub/src/main/ballerina/src/websub/subscriber_service_endpoint.bal

```diff
@@ -39,8 +39,22 @@ public type Listener object {
     #
     # + port - The port number of the remote service
     # + config - The configurations related to the `websub:Listener`
-    public function __init(int port, SubscriberListenerConfiguration? config = ()) {
-        self.init(port, config);
+    # 
+    # Gets called when the `websub:Listener` is being initialized during the module initialization.
```

Comment:
@MaryamZi Can we also remove these documentation lines?

Reply:
Yes this too.

Docs take the following format.

```
    # Description.
    #
    # + p1 - Description for param 1.
    # + p2 - Description for param 2.
    public function init(int param1, string? param2 = ()) {
    }
```

After the params (or fields in a record/object type) you'll only have deprecation documentation.